.travis.yml

```diff
@@ -29,7 +29,7 @@ before_script:
 
   # waiting for the DynamoDB mock
   - |
-    while [[ $(curl -so /dev/null -w '%{response_code}' localhost:4569) != '502' ]]
+    while [[ $(curl -so /dev/null -w '%{response_code}' localhost:4569) != '400' ]]
```

Comment:
Thanks, @Jamim. Could you explain what's happening here? The traditional pattern is to hit some healthcheck endpoint and expect a 2xx response, so checking for 400 still seems odd.

Reply:
but since it works I will merge to unblock other PRs.